console/src/components/UIKit/atoms/Badge/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import StyledBadge, { StyledOwnBadgeProps } from './Badge';
+
+export type AllowedBadges =
+  | ''
+  | 'error'
+  | 'no updates'
+  | 'community'
+  | 'update'
+  | 'event';
```

Comment:
Note: these types should be communicated with the people who would control this.